### PR TITLE
Refactor ui v1.4

### PR DESCRIPTION
**Pull request by bscrow**
Major refactor to HireLah UI layout as well as UI components. 

Modified how Attribute scores are managed in Transcript, it is now using an ObservableList of Pairs.

### CODE REVIEW

File: src/main/java/seedu/address/model/hirelah/Transcript.java [hunk 1]
```diff
@@ -35,15 +33,10 @@ public Transcript(QuestionList questions, AttributeList attributes) {
      */
     public Transcript(QuestionList questions, AttributeList attributes, Instant startTime) {
         this.remarkList = new RemarkList(questions.size(), startTime);
-        this.attributeToScoreMap = FXCollections.observableHashMap();
+        this.attributeToScoreData = FXCollections.observableArrayList();
         this.completed = false;
```
Comment (agnesnatasya) [inline src/main/java/seedu/address/model/hirelah/Transcript.java]: Seems like the attribute to score map works for the first interviewee but it doesnt for the next interviewees. Might have to take a look for this!

Comment (agnesnatasya) [reply src/main/java/seedu/address/model/hirelah/Transcript.java]: The bar chart does not show up in the second interview held!

File: src/main/resources/view/DetailedIntervieweeCard.fxml [hunk 2]
```diff
@@ -1,64 +1,32 @@
...
-<?import javafx.scene.layout.RowConstraints?>
-<?import javafx.scene.layout.StackPane?>
-<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.chart.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<VBox fx:id="detailedIntervieweePane" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1"
+      maxWidth="250" minHeight="490" minWidth="250" prefHeight="490" >
+    <Label fx:id="name" layoutX="20.0" layoutY="10.0" styleClass="cell_big_label" text="\$first" />
```
Comment (agnesnatasya) [inline src/main/resources/view/DetailedIntervieweeCard.fxml]: When there are a lot of attributes, the bar chart does not scroll and it becomes misaligned

File: src/main/java/seedu/address/ui/InterviewPanel.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    }
+
+    /**
+     * Scrolls the Transcript to the given index.
+     *
+     * @param index the index to scroll to.
+     */
+    public void scrollTo(int index) {
+        remarkListPanel.scrollTo(index);
+    }
```
Comment (agnesnatasya) [inline src/main/java/seedu/address/ui/InterviewPanel.java]: The 'goto command' is supposed to scroll the desired remark but it does not work.

Comment (agnesnatasya) [reply src/main/java/seedu/address/ui/InterviewPanel.java]: Screenshot 2020-04-09 at 4 18 04 PM

**Comment by bscrow** [comment]
After some discussions, we have decided to scrap the BarChart to represent Attribute scores for interviewees due to stability issues inherent with BarChart in JavaFX. A tableView will be used instead.

File: src/main/java/hirelah/ui/SessionInformationCard.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    @FXML
+    private Label finaliseStatus;
+
+    public SessionInformationCard() {
+        super(FXML);
+        sessionName.setText("Session: " + "CEO Interview");
+        ObservableValue<Integer> numberInterviewed = new SimpleIntegerProperty(9).asObject();
+        ObservableValue<Integer> numberInterviewees = new SimpleIntegerProperty(20).asObject();
+        ObservableValue<Boolean> isFinalised = new SimpleBooleanProperty(true).asObject();
```
Comment (CornCobs) [inline src/main/java/hirelah/ui/SessionInformationCard.java]: The values here are still hard-coded!

**Review by CornCobs — COMMENTED** [review]
Great job! Once all the UI elements are pointing to real Model objects our app should be fully working and user testable!

**Comment by bscrow** [comment]
Resolved bugs pointed out above. 

Resolves #116, #117

**Review by CornCobs — COMMENTED** [review]
Looks good to me!

**Pull request merged**